key.css

```
@@ -144,14 +141,20 @@ h6{
     justify-content: center;
     align-items: center;
 }
-
+.lives{
+    display: inline-block;
+}
 .heart{
-    height: 50px;
-    width: 50px;
-    color: red;
-}
-.h-broken{
-    color: red;
-    height:45px ;
-    width: 45px;
-}
\ No newline at end of file
+    transform: translateX(-2000%);
+    height: 30px;
+    width: 30px;
+}
+.default{
```

If this is related to lives, then you've to include the word `lives` in it, otherwise, the code becomes unreadable.

```suggestion
.default_lives {
```